docs/source/development/tools/index.rst

```diff
@@ -0,0 +1,4 @@
+.. toctree::
```

Comment:
Looking at the rendered docs: https://imap-processing--163.org.readthedocs.build/en/163/development/tools/xtce_generator.html

It looks like this page is showing up as <no-name> in the nav bar.

I wonder if this needs a title as well so it will show up with nested items like the style-guide entry?

```suggestion
Tools
=====

.. toctree::
```